test/routes/src/server/routes/t_github-auth.js

```diff
@@ -4,9 +4,17 @@ import nock from 'nock';
 import expect from 'expect';
 import url from 'url';
 
-import auth from '../../../../../src/server/routes/github-auth';
 import db from '../../../../../src/server/db';
+import {
+  listOrganizationsCacheId
+} from '../../../../../src/server/handlers/github';
```

Comment:
FYI We talked recently in the fortnightly JS discussion about moving tests along side src as a way to resolve this `../../../../../` line noise, something for the future maybe, but maybe we should open that meeting up to members of OLS and further afield.